### ISSUE TITLE
picture in scrimba brokwn

### ISSUE THREAD

**Issue by lredmo1** [learn-co-curriculum/react-hooks-information-flow-code-along#3]
### Canvas Link

https://learning.flatironschool.com/courses/4385/modules/items/310712

### Concern

The module page with the Scrimba doesn't have a flag icon to report new issues on so I am reporting it here, on the page that follows in the State and Events module. In the first video of Scrimba Playlist: Thinking in React, at about 1 min 30 sec left, the speaker pastes in a URL or file path for a picture in that does not render in the console. He references this picture throughout the rest of the video to explain things and into the next but the picture isn't showing up (alt text shows instead). Video is hard to follow without seeing the picture he is referencing.

**Comment by ihollander** [comment]
@lredmo1 Thanks for flagging! It looks like the React documentation has changed since this video was recorded and that image is no longer available. When we rerecord the video we'll make sure to use an updated image link. In the meantime, I've updated the video description in Scribma to include a new reference image link.

![thinking in react](https://reactjs.org/static/9381f09e609723a8bb6e4ba1a7713b46/90cbd/thinking-in-react-components.png)
